applications/FluidDynamicsApplication/custom_processes/shock_capturing_physics_based_process.h

```diff
@@ -331,7 +335,12 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) ShockCapturingPhysicsBasedProcess :
                 const double Pr_beta_min = 0.9;
                 const double alpha_pr_beta = 2.0;
                 const double mach_threshold = 3.0;
-                const double Pr_beta = Pr_beta_min * (1.0 + std::exp(-2.0 * alpha_pr_beta * (mach - mach_threshold)));
+                double Pr_beta;
```

Comment:
I do not know if you want to include aesthetic changes here, but this should be `pr_beta`

Reply:
As it refers to the Prandtl number, which is commonly referred in the literature as Pr, I'd leave this particular one as it is.

Reply:
ok, that might make it more clear.

Reply:
Instead we can name it `prandtl_beta`, which would follow the style guide and be as clear. Up to you @Marco1410 and @ddiezrod.